### DISCUSSION TITLE
I cannot install pybricksdev

### DISCUSSION THREAD

**Discussion by kajr0s** [pybricks/support#1413] [Q&A] [answered]
Windows 11
python 3.12.1

pip install pybricksdev

The conflict is caused by:
pybricksdev 1.0.0a17 depends on winrt<2.0.0 and >=1.0.21033; sys_platform == "win32"
pybricksdev 1.0.0a16 depends on winrt<2.0.0 and >=1.0.21033; sys_platform == "win32"
pybricksdev 1.0.0a15 depends on winrt<2.0.0 and >=1.0.21033; sys_platform == "win32"
pybricksdev 1.0.0a14 depends on winrt<2.0.0 and >=1.0.21033; sys_platform == "win32"
pybricksdev 1.0.0a13 depends on winrt<2.0.0 and >=1.0.21033; sys_platform == "win32"
pybricksdev 1.0.0a12 depends on winrt<2.0.0 and >=1.0.21033; sys_platform == "win32"
pybricksdev 1.0.0a11 depends on winrt<2.0.0 and >=1.0.21033; sys_platform == "win32"
pybricksdev 1.0.0a10 depends on winrt<2.0.0 and >=1.0.21033; sys_platform == "win32"

To fix this you could try to:

loosen the range of package versions you've specified
remove package versions to allow pip attempt to solve the dependency conflict

The winrt package is missing on windows?

Does this mean that pybricksdev does not work on windows?

**Comment by dlech** [comment]
It looks like it is trying to install a very old version. The current version is `1.0.0a46`. So maybe `pip install pybricksdev==1.0.0a46` would work?

**Comment by kajr0s** [comment, marked as answer]
Oh... It seems that pybricksdev requires python 3.11, not 3.12... Thanks

**Reply by datgame** [reply]
This fixed it for me too.
Weird that this issue is closed, as it's still a problem and 3.12 is the recommended version on python.org, which pybricks recommends you go to.
The pybricks docs should at least say you should get 3.11 until the problem is fixed.

**Reply by dlech** [reply]
This is a discussion, not an issue. But feel free to open an issue if it is causing problems.
